pages/content.py

```diff
@@ -10,6 +10,7 @@
 class Content(Base):
     _content_locator = (By.ID, 'content')
     _title_locator = (By.CSS_SELECTOR, '.media-title h1')
+    _ncy_locator = (By.CSS_SELECTOR, 'not-converted-yet')
```

Comment:
If it's a class you need to add the `.` at the beginning. So this should be `.not-converted-yet`

Reply:
I also recommend not using the CSS selector but actually the By.CLASS_NAME in this case. I think CSS selectors should be a last resort really. I've used them so much b/c there has been no other way. In an ideal world we would have the devs add ids.